metadata-ingestion/src/datahub/ingestion/source/openapi.py

```diff
@@ -52,6 +52,7 @@ class OpenApiConfig(ConfigModel):
     ignore_endpoints: list = Field(default=[], description="")
     username: str = Field(default="", description="")
     password: str = Field(default="", description="")
+    proxies: dict = Field(default={"http": None, "https": None}, description="")
```

Comment:
given that we use `proxies: Optional[dict] = None` everywhere, is there a reason we need the default here instead of leaving the default as None?

We should also add the config format to the description, so that it shows up in our docs